database/layer.go

```diff
@@ -131,6 +131,59 @@ func InsertLayer(layer *Layer) error {
 	return nil
 }
 
+// DeleteLayer delete the specified layer and every layers that are based on that one, recursively.
```

Comment:
Dang it. Such a pain in the butt to switch back and forth between 100C (Python) to 80C (Go).